python/lsst/ap/verify/ap_verify.py

```diff
@@ -188,7 +190,7 @@ def runApVerify(cmdLine=None):
         or 127 if the task runner framework failed.
     """
     lsst.log.configure()
```

Comment:
I believe this call and the one in `runIngestion` are what should be replaced with any `logging.basicConfig` and `lsst.log` forwarding that is needed. These two functions should be the only (non-test) entry points to the code.